.github/workflows/periodic-integration-tests.yml

```diff
@@ -6,15 +6,80 @@ on:
   workflow_dispatch:
```

Comment:
TODO: Figure out if we need to specify a list of secrets like pre-merge-CI (`integration-tests.yml`), maybe chat with Kenny on this

Reply:
You shouldn't have to since this is a standalone job.